src/eva/vision/data/transforms/common/resize_and_clamp.py

```diff
@@ -37,15 +38,15 @@ def __init__(
     def _build_transforms(self) -> Sequence[Callable]:
         """Builds and returns the list of transforms."""
         transforms = [
-            torch_transforms.Resize(size=self._size),
-            torch_transforms.CenterCrop(size=self._size),
+            v2.Resize(size=self._size),
+            v2.CenterCrop(size=self._size),
+            v2.ToDtype(torch.float32),
             normalization.Clamp(out_range=self._clamp_range),
-            torch_transforms.ToDtype(torch.float32),
             normalization.RescaleIntensity(
                 in_range=self._clamp_range,
                 out_range=(0.0, 1.0),
             ),
-            torch_transforms.Normalize(
+            v2.Normalize(
```

Comment:
warning here, `Clamp` should be before `ToDtype`, as clamp values refer to the image pixel int values -- with this change, `TotalSegmentator` will not be normalized correctly 

we can to change the order

Reply:
done